src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow
```

Comment:
I just pinged the flow team and `@flow weak` is described as a hack, very experimental and not recommended because it keeps errors latent. We have less than 20 files in the entire codebase using it. So, we should not using it on React :)